cmd/contour/serve.go

```diff
@@ -361,10 +389,8 @@ type informer interface {
 
 func startInformer(inf informer, log logrus.FieldLogger) func(stop <-chan struct{}) error {
 	return func(stop <-chan struct{}) error {
-		log.Println("waiting for cache sync")
-		inf.WaitForCacheSync(stop)
 
-		log.Println("started")
+		log.Println("starting informer")
```

Comment:
suggestion: maybe the next line should say "stopped informer", but that might be redundant as the context is already `context=coreinformers`